corehq/motech/repeaters/migrations/0002_sqlrepeatrecord.py

```diff
@@ -0,0 +1,113 @@
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('repeaters', '0001_adjust_auth_field_format'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='SQLRepeaterStub',
+            fields=[
+                ('id', models.AutoField(auto_created=True,
+                                        primary_key=True,
+                                        serialize=False,
+                                        verbose_name='ID')),
```

Comment:
This is much prettier formatting that the default generated by the `makemigrations` command, but something is sticking in my memory that Django will want to reformat this file each time `makemigrations` is run on this app. Is that true, or am I mis-remembering?  (I'd be happy I'm wrong)

Reply:
This should be okay - running `makemigrations` on future changes won't affect the formatting of existing migrations.

Reply:
Good question! I tested. When Django creates a new migration, it does not touch existing migrations.